### DISCUSSION TITLE
Source Members not Showing

### DISCUSSION THREAD

**Discussion by HenryMiller** [codefori/vscode-ibmi#654] [General]
Issue text goes here.
<img width="309" alt="Screenshot at Apr 26 17-38-00" src="https://user-images.githubusercontent.com/1829995/165339624-f544be81-dd46-466c-99c0-8a4d66990298.png">

* QCCSID: 65535
* Features enabled:
   * git: true
   * grep: true
   * tn5250: true
   * setccsid: true
   * GENCMDXML.PGM: false
   * QZDFMDB2.PGM: true
   * iconv: true
   * attr: true
* SQL enabled: false
* Source dates enabled: false

**Comment by worksofliam** [comment]
Hi Henry. Change your filter to use `*SRCPF` in the object type.

**Reply by HenryMiller** [reply]
Hi Barry,
thank you. it works now!
